.github/workflows/ci.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10, 12, 14, 15]
+        node: [12, 14, 15]
```

Comment:
I understand why we want to do this - I am guessing there is a section in the README that says it supports Node 10 though, and I'm guessing there's a section that talks about legacy tags for node.js 8.x.  Right?  We probably need to rationalize these.